libyul/backends/evm/EVMDialect.cpp

```diff
@@ -124,6 +124,17 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 		)
 			builtins.emplace(createEVMFunction(instr.first, instr.second));
 
+	builtins.emplace(createFunction("linkersymbol", 1, 1, SideEffects{}, {true}, [](
```

Comment:
I went with `linkersymbol()` rather than `linkerSymbol()` (as in EVM assembly) to follow the convention used for other Yul builtins. But maybe I should have used `linkerSymbol()` for consistency with assembly instead? What do you think?

Reply:
everything lowercase better fits the other yul builtins